app/main/helpers/validation.py

```diff
@@ -96,6 +98,16 @@ def character_limit_errors(self):
 
         return errors_map
 
+    def word_limit_errors(self):
+        errors_map = {}
+        for question_id in self.all_fields():
+            if self.content.get_question(question_id).get('type') in ['text', 'textbox_large']:
+                answer = self.answers.get(question_id) or ''
+                if self.word_limit is not None and len(answer.split(" ")) > self.word_limit:
+                    errors_map[question_id] = "under_word_limit"
```

Comment:
I'm confused, why is this error called `under_word_limit`?

Reply:
It matches the existing `under_character_limit`.

Reply:
Any chance we can change both to `over_*_limit`? Or just `*_limit`?